lib/instance.js

```diff
@@ -642,14 +643,14 @@ Instance.prototype.save = function(options) {
 
       if (self.isNewRecord) {
         query = 'insert';
-        args = [self, self.Model.getTableName(options), values, options];
+        args = [self, self.Model.getTableName(this.$options), values, options];
       } else {
         var where = this.where();
 
         where = Utils.mapValueFieldNames(where, Object.keys(where), this.Model);
 
         query = 'update';
-        args = [self, self.Model.getTableName(options), values, where, options];
+        args = [self, self.Model.getTableName(this.$options), values, where, options];
```

Comment:
Same goes for the rest of the codebase where call options has been changed for model/instance options (which doesn't make a lot of sense to me actually, why would instance options be used?)